common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -1164,27 +1160,27 @@ static int hashCodeAsciiSafe(byte[] bytes, int startPos, int length) {
             hash = PlatformDependent0.hashCodeAsciiCompute(getLongSafe(bytes, i), hash);
         }
         switch(remainingBytes) {
-        case 7:
-            return ((hash * HASH_CODE_C1 + hashCodeAsciiSanitize(bytes[startPos]))
```

Comment:
@kezhenxu94 can you please revert all the "formatting-only" changes ?

Reply:
DONE